_posts/2020-27-11-eol.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  "EOL of EL6 and EL7 and removal Customer portal support"
+date:   2020-11-27 12:00:00 +0100
+author: Miroslav Suchý
+visible: 1
+categories:
+---
+
+## Packages will not be built for EL6 and EL7.
+
+For [long](https://github.com/abrt/faf/issues/735) [time](https://github.com/abrt/abrt/issues/1337), we talked about EOLing EL6 and EL7. What it actually means?
+
+RHEL 6 is [going to be EOL](https://access.redhat.com/discussions/2399461) at the end of this month (30th November 2020). We will no longer build ABRT packages for EL6 and we will stop supporting EL6 as content.
+
+RHEL 7 is still active but quite old. ABRT team will stop testing, building, and developing on top of RHEL7. We are going to focus on RHEL 8 and upcoming RHEL and Fedora. We still support RHEL 7 as content (e.g., in ABRT Analytics).
+
+This is a description of the current de-facto state. The last release of ABRT Analytics was not already build for RHEL 7.
+
+## End of reporting to Customer Portal
+
+Red Hat Customer Portal migrated to a new backend that does not have the API which we need and as result of that we removed reporting to Customer Portal from ABRT. 
```

Comment:
This is a small nitpick, but it was not removed directly from ABRT but from libreport. For example Anaconda that uses libreport directly will  also  be affected even if they do not use ABRT.